rls/deploy/deploy_cid_rls.yaml

```
@@ -18,6 +21,14 @@ Parameters:
     Type: String
     AllowedPattern: ([a-z0-9\-, ]*?$)
     Description: "(Ex: 123456789,098654321,789054312) List of Payer IDs you wish to collect data for. Can just be one Accounts"
+  DatabaseName:
+    Type: String
+    Description: Athena Databse where to deploy RLS table, should be same as CID Cudos database
```

Need to say existing?
```suggestion
    Description: Existing Athena Database where to deploy RLS table, should be same as CID database
```